src/main/java/com/google/api/generator/gapic/composer/SampleCodeHelperComposer.java

```diff
@@ -115,15 +115,32 @@ private static TryCatchStatement composeLroUnaryRpcMethodSampleCode(
   }
 
   private static TryCatchStatement composePagedUnaryRpcMethodSampleCode(
-      Method method, List<MethodArgument> arguments, TypeNode clientType) {
-    // TODO(summerji): compose sample code for unary paged rpc method.
-    // TODO(summerji): Add unit tests.
+      Method method,
+      List<MethodArgument> arguments,
+      TypeNode clientType,
+      Map<String, ResourceName> resourceNames) {
+    // TODO(summerji): Add unit test.
+    // Assign each method arguments with default value.
+    List<Statement> bodyStatements =
+        arguments.stream()
+            .map(
+                methodArg ->
+                    ExprStatement.withExpr(
+                        assignMethodArgumentWithDefaultValue(methodArg, resourceNames)))
+            .collect(Collectors.toList());
+    // For loop client on iterateAll method.
+    // e.g. for (LoggingServiceV2Client loggingServiceV2Client :
+    // loggingServiceV2Client.ListLogs(parent).iterateAll()) {
+    // //doThingsWith(element);}
+    bodyStatements.add(
+        ForStatement.builder()
+            .setLocalVariableExpr(createVariableDeclExpr(getClientName(clientType), clientType))
```

Comment:
Instead of creating this variable twice, let's just do it once and reuse it in both the for loop and initial variable declaration.

Reply:
Will do.